config/v1/0000_10_config-operator_01_image.crd.yaml

```diff
@@ -122,6 +122,16 @@ spec:
                   type: array
                   items:
                     type: string
+                unqualifiedSearchRegistries:
+                  description: 'unqualifiedSearchRegistries are registries that will
+                    be searched when pulling images that do not have fully qualified
+                    domains in their pull specs. Registries will be searched in the
+                    order provided in the list. Note: this search list does NOT work
+                    with builds and imagestream imports. It only works with the container
+                    runtime, CRI-O.'
```

Comment:
i wonder if we want to pick a different name here, because if we ever did want to support search for imagestreams+builds, we would have to add a new field.  We couldn't just suddenly change how builds+imagestreams do shortname resolution.

If we named this field in a way that made it clear it only applied to crio pulls, then if we wanted to add build+imagestream support in the future, we could introduce this field name for that, taking precedence over the old field.

does that make sense?

Reply:
yup makes sense. Something like `RuntimeShortNameRegistries` or `RuntimeSearchRegistries` or `RuntimeShortNamePullRegistries`? If not runtime we could use crio, wdyt?

Reply:
RuntimeSearchRegistries could work or maybe NodeSearchRegistries since it is affecting image specs consumed by the node?

Reply:
Both sound good to me, no particular preference. @mrunalp wdyt?

Reply:
How about `ContainerRuntimeSearchRegistries`?

Reply:
> How about ContainerRuntimeSearchRegistries?

i like it

Reply:
Fixed! PTAL :)